cranelift-codegen/src/isa/x86/unwind.rs

```diff
@@ -0,0 +1,525 @@
+//! Unwind information for x64 Windows.
+use super::registers::RU;
+use crate::ir::{Function, InstructionData, Opcode};
+use crate::isa::{CallConv, RegUnit, TargetIsa};
+use alloc::vec::Vec;
+
+// Maximum (inclusive) size of a "small" stack allocation
+const SMALL_ALLOC_MAX_SIZE: u32 = 128;
+// Maximum (inclusive) size of a "large" stack allocation that can represented in 16-bits
```

Comment:
Please use doc comments here and above.

Reply:
I generally don't use doc comment style for privates (I like it to warn/error if suddenly these get exposed as public since the comment might not be "up to par" for public documentation), but if that's the convention for Cranelift, I will fix.

Reply:
I dont now if it is convention, but it is useful when using the rustdoc option to document private items.